modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -437,6 +437,14 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 			_objectRelationship.getObjectRelationshipId(),
 			objectEntry8.getObjectEntryId());
 
+		// Remove object entry to assert empty one to many relationship
+
+		_updateObjectEntry(
+			objectEntry7.getObjectEntryId(),
+			HashMapBuilder.<String, Serializable>put(
+				_relationshipObjectField.getName(), 0
```

Comment:
@G-Sa In the bug ticket description we have:

> 5. Edit the entry for B and try to remove the relationship by leaving it empty.

Shouldn't we pass an empty string instead?

Reply:
What i understand from that line is that we should remove the relationship between the two entries, as in allowing the object entry for B to not be related to anything anymore, thats why i edited the field assigning it to "nothing"
@carolmariaabb

Reply:
Ok, thanks @G-Sa